.github/workflows/sycl-linux-build.yml

```diff
@@ -167,7 +167,8 @@ jobs:
           --cmake-opt=-DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
           --cmake-opt="-DLLVM_INSTALL_UTILS=ON" \
           --cmake-opt="-DNATIVECPU_USE_OCK=Off" \
-          --cmake-opt="-DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=SPIRV"
+          --cmake-opt="-DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=SPIRV" \
+          --level_zero_v1_and_v2
```

Comment:
option name is a bit weird, any other ideas?

Reply:
I tried to keep it similar to the other one that enables v2 only, but I'm open for suggestions.

Reply:
yeah i got nothing

Reply:
@sarnex I reworked this in this PR: https://github.com/intel/llvm/pull/17407. You might want to take a look.